src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -655,10 +655,12 @@ machine(MachineType:TCC, "TCC Cache")
 
   action(p_profileMiss, "pm", desc="Profile cache miss") {
       L2cache.profileDemandMiss();
+      DPRINTF(RubyHitMiss, "in TCC miss at %p \n", address);
```

Comment:
A while (years) back we had some changes to make Ruby address prints be consistent.  I believe we decided on `%#lx`.  It makes it easier to search through ProtocolTrace output